### PR TITLE
Created proper Test Cases for Arbitrary Files Repo and Content View s…

### LINKED ISSUE #5088: Fix Arbitrary File TestCases
Stubs are not on correct classes, they should be placed on their own classes.

### PR DESCRIPTION
**Pull request by renzon**
…tubs

close #5088

@svtkachenko I read your comments on irc and created this PR because #5053 was already merged.

### CODE REVIEW

**Comment by sghai** [comment]
ACK

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -1808,6 +1808,11 @@ def test_positive_publish_promote_with_RH_ostree_and_other(self):
         self.assertEqual(
             len(content_view.read().version[0].read().environment), 2)
 
+
+class ContentViewFileRepo(APITestCase):
```
Comment (svtkachenko) [inline tests/foreman/api/test_contentview.py]: Perhaps `ContentViewFileRepoTestCase`?

Comment (oshtaier) [inline tests/foreman/api/test_contentview.py]: that's for sure, not perhaps :)

**Comment by sghai** [comment]
Merging as @svtkachenko has approved the changes..

**Pull request merged**